doc/sphinx/source/recipes/recipe_climwip.rst

```diff
@@ -0,0 +1,131 @@
+.. _recipe_climwip:
+
+ClimWIP: independence & performance weighting
+=============================================
+
+Overview
+--------
+
+This recipe calculates weights based on combined performance and independence metrics. These weights can be used in subsequent diagnostics.
+Reference implementation based on https://github.com/lukasbrunner/ClimWIP
+
+
+Available recipes and diagnostics
+---------------------------------
+
+Recipes are stored in esmvaltool/recipes/
+
+    * ``recipe_climwip.yml``
+
+Diagnostics are stored in esmvaltool/diag_scripts/weighting/
+
+    * ``climwip.py``: Compute weights for each input dataset
+    * ``weighted_temperature_graph.py``: Show the difference between weighted and non-weighted temperature anomalies.
+
+
+User settings in recipe
+-----------------------
+
+1. Script ``climwip.py``
+
+  *Required settings for script*
+    * ``sigma_performance``: shape parameter weights calculation (determined offline)
+    * ``sigma_independence``: shape parameter for weights calculation (determined offline)
+    * ``obs_data``: list of project names to specify which are the the observational data. The rest is assumed to be model data.
```

Comment:
The formatting of this block looks very different on readthedocs than the other blocks (click the 'details' button behind the readthedocs build to see the preview). I think this looks actually nicer, would it be possible to also format the other blocks like this? If not, it would be best to use a consistent style for all blocks.

Reply:
Wow, that preview option is super useful! Thanks!! 
I didn't notice the layout had changed, I'll look into it.

Reply:
Ah, my local build preview is different from the version on github, that's why I didn't notice. Now I'll have to wait to see the GH actions build.

Reply:
Done!